lib/capistrano/tasks/stretcher.rake

```diff
@@ -5,6 +5,7 @@ require 'yaml'
 namespace :load do
   task :defaults do
     set :gzip_compress_level, ""
```

Comment:
Like this ? 

```ruby
set :archiver_option ""
set :gzip_archier "gzip"
```

If we want to specify additional options,  `:archiver_option` seems to be better semantic.   ( e.g. `pigz` has some options to tune perfomance )

```
       -b --blocksize mmm
              Set compression block size to mmmK (default 128KiB).

       -p --processes n
              Allow up to n processes (default is the number of online processors)
```

e.g. 

```ruby
set :archiver_option "--best -b 1024 -p 24 "
set :gzip_archier "pigz"
```